### DISCUSSION TITLE
Faunadb example update

### DISCUSSION THREAD

**Discussion by vorcigernix** [vercel/next.js#37647] [Show and tell] [answered]
I decided to use faunadb for my new project and I had to update + rewrite swr part slightly to use the current versions of technologies. How do I patch the example code on vercel? Should I submit a new example as described in the contributing doc?

[Code is here](https://github.com/vorcigernix/zitrone/tree/with-faunadb).

**Comment by leerob** [comment, marked as answer]
Hey! If you'd like to open a PR to update the example, that would be amazing 😄 

https://github.com/vercel/next.js/blob/canary/contributing.md

**Comment by vorcigernix** [comment]
Okay, thx, hope I did it right.
